src/components/Navigation/Pages/PageBrowse/ButtonsBar.js

```diff
@@ -21,9 +21,11 @@ export default class ButtonsBar extends Component {
             <span>&nbsp;Revert Changes</span>
           </Button>
         </ButtonWithTooltip>
-        <Button bsStyle="default" onClick={toggleCollapseExpandAll}>
-          Toggle Collapse
-        </Button>
+        {toggleCollapseExpandAll && (
```

Comment:
Is `toggleCollapseExpandAll` not always defined?

Reply:
toggleCollapseExpandAll is changed to be undefined in PageBrowse.renderButtons.